packages/migrate-config/tests/fixtures/plugins-dedupe/.eslintrc.yml

```diff
@@ -0,0 +1,5 @@
+root: true
+plugins:
+    - import
+extends:
+    - plugin:import/errors
```

Comment:
This test doesn't seem to be representative for this change, because the output is the same before and after the change.

I think we'd need an example where one plugin needs the fixup while the other doesn't, for example:

```yaml
root: true
plugins:
    - import
    - n
extends:
    - plugin:import/errors
    - plugin:n/recommended

```

Reply:
Ah good call.